### ISSUE TITLE
Tutorial's JavaScript code example

### ISSUE THREAD

**Issue by jeremykohn** [HappenApps/Quiver#604]
Here's the JavaScript from step 2 of the tutorial:

```
// This is a code cell set to the JavaScript mode

void hello()
{
    console.log("Hello World!");
}
```

Corrected version:

```
// This is a code cell set to the JavaScript mode

function hello() {
    console.log("Hello World!");
}
```

Not really a "bug", though I think it should still be fixed.

**Comment by ylian** [comment]
Good catch. Thanks.

**Comment by ylian** [comment]
Fixed in 3.0.3. Need to click "Help > Show Tutorial" to reload the tutorials.
